pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1143,7 +1143,7 @@
         "4.1.0.0": "4.1.0",
         "4.1.1.0": "4.3.0",
         "4.2.0.0": "4.4.0",
-        "4.2.1.0": "4.5.0"
+        "4.3.0.0": "4.5.0"
```

Comment:
@ericstj to confirm but I think we should bump the version to 4.3.0.0 given we are adding new APIs.